src/_scss/pages/interactiveDataSources/_scroller.scss

```diff
@@ -0,0 +1,39 @@
+$top: 60px;
+
+.scroller-container {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.sk-scroller-content-collection {
```

Comment:
Looks like an `sk-` made it in. Probably worth a `grep` or find all to make sure you've caught all of them

Reply:
So I'm only going to remove it from `_scroller.scss` even though I also found it in `_downloadBottomBar.scss` as shown here: https://github.com/fedspendingtransparency/usaspending-website/blob/91a8d9a5f2f8f5bf20f3ea36289c6ca6d8f18e3e/src/_scss/layouts/default/footer/_downloadBottomBar.scss#L87
It seems to have already existed in `_downloadBottomBar.scss`

Reply:
Ha that's funny, idk that what is. As long as it's not in any code that we're _adding_ we're gtg